adhocracy-plus/assets/scss/_form.scss

```diff
@@ -153,8 +155,17 @@ div.cke_focus {
 //kept from before bootstrap for datepicker
 .input-group__input {
     flex: 1 1 auto;
+    box-shadow: none;
 }
 
 .input-group__after {
     flex: 0 1 0%;
 }
+
+.dropdown > button {
+    &:hover,
+    &:focus {
+        outline: none;
+        box-shadow: none;
+    }
+}
```

Comment:
this is here and not in dropdown because linter had order issue and wouldn't accept a reorder but I have it in my notes to fix, I think most of dropdown can be cleaned but can't do it tonight!